packages/amplify-graphql-model-transformer/src/graphql-types/common.ts

```diff
@@ -174,6 +177,25 @@ const addDatastoreConditions = (input: InputObjectDefinitionWrapper, ctx: Transf
   }
 };
 
+/**
+ * Add implicit timestamp fields createdAt and updatedAt automatically included in all models
+ * @param input output object
+ */
+const addTimestampFields = (input: InputObjectDefinitionWrapper, isSubscriptionFilter: boolean) => {
+  const timestampFields = [
```

Comment:
Do we consider the case where model directive definition contains a custom timestamp fields?